x-pack/filebeat/input/awss3/collector.go

```diff
@@ -21,12 +21,15 @@ import (
 	"sync"
 	"time"
 
+	"github.com/elastic/beats/v7/libbeat/statestore"
```

Comment:
this should be moved down around line 34

Reply:
I have a file watcher on goland that should take care of ordering imports, that sometimes fails
but I would expect mage update would take care of this as well